src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -27,29 +28,34 @@ import * as Link from '../../libs/actions/Link';
 const propTypes = {
     /** Bank account currently in setup */
     reimbursementAccount: reimbursementAccountPropTypes.isRequired,
-
+    onfidoToken: PropTypes.string,
```

Comment:
`onfidoToken` is undefined when we first get to this step because we haven't yet added the personal information